Sources/Talkshoplive/Core/Show/Show.swift

```diff
@@ -45,9 +46,34 @@ public class Show {
     public func getStatus(showKey: String, completion: @escaping (Result<EventData, Error>) -> Void) {
         ShowProvider().fetchCurrentEvent(showKey: showKey) { result in
             switch result {
-            case .success(let apiResponse):
+            case .success(let eventInstance):
                 // Set the details and invoke the completion with success.
-                completion(.success(apiResponse))
+                completion(.success(eventInstance))
+                if self.incrementedView[showKey] == nil {
+                    self.incrementedView[showKey] = false
+                }
+                if self.incrementedView[showKey] == false ,
```

Comment:
Can't we just check for the `nil` condition and increment? Why do we have to assign `false` here first?

Reply:
@daman-talkshoplive I appreciate your suggestion! Initializing with false provides a structured approach, fostering code clarity and consistency for smoother collaboration. I will do needful changes as suggested.